libs/category/driver/magento/src/category.service.ts

```diff
@@ -63,9 +64,9 @@ export class DaffMagentoCategoryService implements DaffCategoryServiceInterface
   /**
```

Comment:
Looks like the todo on line 63 can be removed

Reply:
removed